README.md

```diff
@@ -1,105 +1,201 @@
+publishing, and versioning guidance.
 
-If you are new, there's also a simpler introduction.  See the [Hello World JavaScript Action](https://github.com/actions/hello-world-javascript-action)
+If you are new, there's also a simpler introduction in the
+[Hello world JavaScript action repository](https://github.com/actions/hello-world-javascript-action).
 
-## Create an action from this template
+## Create Your Own Action
 
-Click the `Use this Template` and provide the new repo details for your action
+To create your own action, you can use this repository as a template! Just
+follow the below instructions:
 
-## Code in Main
+1. Click the **Use this template** button at the top of the repository
+1. Select **Create a new repository**
+1. Select an owner and name for your new repository
+1. Click **Create repository**
+1. Clone your new repository
 
-> First, you'll need to have a reasonably modern version of `node` handy. This won't work with versions older than 9, for instance.
+## Initial Setup
 
-Install the dependencies  
-```bash
-$ npm install
-```
+After you've cloned the repository to your local machine or codespace, you'll
+need to perform some initial setup steps before you can develop your action.
 
-Build the typescript and package it for distribution
-```bash
-$ npm run build && npm run package
-```
+> [!NOTE]
+>
+> You'll need to have a reasonably modern version of
+> [Node.js](https://nodejs.org) handy. If you are using a version manager like
+> [`nodenv`](https://github.com/nodenv/nodenv), you can run `nodenv install` in
+> the root of your repository to install the version specified in
+> [`.node-version`](.node-version). Otherwise, 16.x or later should work!
 
-Run the tests :heavy_check_mark:  
-```bash
-$ npm test
+1. :hammer_and_wrench: Install the dependencies
 
- PASS  ./index.test.js
-  ✓ throws invalid number (3ms)
-  ✓ wait 500 ms (504ms)
-  ✓ test runs (95ms)
+   ```bash
+   npm install
+   ```
 
-...
-```
+1. :building_construction: Package the TypeScript for distribution
+
+   ```bash
+   npm run bundle
+   ```
+
+1. :white_check_mark: Run the tests
+
+   ```bash
+   $ npm test
 
-## Change action.yml
+   PASS  ./index.test.js
+     ✓ throws invalid number (3ms)
+     ✓ wait 500 ms (504ms)
+     ✓ test runs (95ms)
 
-The action.yml defines the inputs and output for your action.
+   ...
+   ```
 
-Update the action.yml with your name, description, inputs and outputs for your action.
+## Update the Action Metadata
 
-See the [documentation](https://help.github.com/en/articles/metadata-syntax-for-github-actions)
+The [`action.yml`](action.yml) file defines metadata about your action, such as
+input(s) and output(s). For details about this file, see
+[Metadata syntax for GitHub Actions](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions).
 
-## Change the Code
+**TODO:** Update `action.yml` with the name, description, inputs, and outputs
```

Comment:
tiny nit: can we remove the `TODO` here? Just to make it more clear that this is something required of the developer instead of repo maintenance 😄

Reply:
Hah, I was a bit torn on that one, and yes I will remove that ;)